src/inflators/pdf.ts

```diff
@@ -0,0 +1,33 @@
+import { addComponent } from "bitecs";
+import { PDFDocumentProxy } from "pdfjs-dist";
+import { Mesh, MeshBasicMaterial } from "three";
+import { HubsWorld } from "../app";
+import { MediaPDF, Networked, NetworkedPDF } from "../bit-components";
+import { PDFResourcesMap } from "../bit-systems/pdf-system";
```

Comment:
I think we generally don't want to be leaking "system state" outside of the system like this. We were previously using `MediaPDF.map` for this (and we actually are still defining that). I think I like that better for this, since it IS component data, it just happens to contain references, so we have to store it a special way. (A good hint of that is that an inflator is writing to it).

I think of "system state" as incidental stuff a system needs to do its jobs, but is localized to that system.

Reply:
It's not "system state" -- It's just wrapping `MediaPDF.map` with type info because it's better to write
`PDFResourcesMap.get(eid).pdf.numPages` than `((MediaPDF as any).map as Map<EntityID, PDFResources>).pdf.numPages`:

```ts
// TODO This can go away if we make bit-components becomes typescript file
export const PDFResourcesMap = (MediaPDF as any).map as Map<EntityID, PDFResources>;
```

Reply:
Oh I missed this. That makes sense. Just a side effect of bit-components not being typescript yet.